wgengine/netstack/netstack.go

```diff
@@ -393,14 +383,15 @@ func (ns *Impl) acceptTCP(r *tcp.ForwarderRequest) {
 		// ForwarderRequest: &{{{{0 0}}} 0xc0001c30b0 0xc0004c3d40 {1240 6 true 826109390 0 true}
 		ns.logf("[v2] ForwarderRequest: %v", r)
 	}
-	var isTailscaleIP bool
 	reqDetails := r.ID()
 	dialAddr := reqDetails.LocalAddress
+	dialNetAddr, _ := netaddr.FromStdIP(net.IP(dialAddr))
```

Comment:
you want to bail out early if this conversion fails?

Reply:
I figured it shouldn't fail, given that we came this far (since there was a similar conversion done for this TCP request at the call of addSubnetAddress).